biz.aQute.bnd.remote.junit/src/aQute/bnd/remote/junit/JUnitFramework.java

```diff
@@ -494,50 +516,40 @@ private boolean isMultiple(Type type) {
 			return ((Class<?>) type).isArray();
 		}
 		if (type instanceof ParameterizedType) {
-
 			Type rawType = ((ParameterizedType) type).getRawType();
 			if (rawType instanceof Class) {
 				Class<?> clazz = (Class<?>) rawType;
 				if (Iterable.class.isAssignableFrom(clazz))
 					return true;
 			}
-
 		}
 		return false;
 	}
 
 	private boolean isParameterizedType(Type to, Class<?> clazz) {
-		if (to instanceof ParameterizedType) {
-			if (((ParameterizedType) to).getRawType() == clazz)
-				return true;
-		}
-		return false;
+		return to instanceof ParameterizedType && ((ParameterizedType) to).getRawType() == clazz;
```

Comment:
I completely agree that the PR became effectively big due to cleanups as I wanted my static analyzers not to complain about the small things in the code. That's how the changes got accumulated. Next time I will definitely concentrate only on the new features instead of making the static analyzer happy 😃